src/main/java/spoon/reflect/declaration/CtElement.java

```diff
@@ -391,4 +393,18 @@
 	 */
 	List<CtElement> getDirectChildren();
 
+	/**
+	 * @return a debug version of the source code of this element, with fully-qualified types. Warning, some implicit elements are not shown in the string.
```

Comment:
> Warning, some implicit elements are not shown in the string.

this warning is probably not true, because DJPP is run in ignoreImplicit=true mode. So everything should be fully qualified when toString is called.
Or do you know a case when implicitness is not ignored?

Reply:
ok, the `this.` qualifier is an example of implicit element. So the warning is correct for me.